conformance/src/main/scala/za/co/absa/enceladus/conformance/CmdConfig.scala

```diff
@@ -40,7 +41,7 @@ case class CmdConfig(datasetName: String = "",
 
 object CmdConfig {
 
-  def getCmdLineArguments(args: Array[String]): CmdConfig = {
+  def getCmdLineArguments(args: Array[String])(implicit spark: SparkSession): CmdConfig = {
```

Comment:
Well.. in case the credentials file is located on HDFS, we're checking its existence.

Seems better to use existing and properly configured HDFS client from spark than trying to load from environment variables?